src/Components/Patient/UpdateStatusDialog.tsx

```diff
@@ -69,9 +101,31 @@ const updateStatusReducer = (state = initialState, action: any) => {
   }
 };
 
+const LinearProgressWithLabel = (props: any) => {
+  return (
+    <Box display="flex" alignItems="center">
+      <Box width="100%" mr={1}>
+        <LinearProgress variant="determinate" {...props} />
+      </Box>
+      <Box minWidth={35}>
+        <Typography variant="body2" color="textSecondary">{`${Math.round(
+          props.value
+        )}%`}</Typography>
+      </Box>
+    </Box>
+  );
+};
+
 const UpdateStatusDialog = (props: Props & WithStyles<typeof styles>) => {
   const { sample, handleOk, handleCancel, classes, userType } = props;
   const [state, dispatch] = useReducer(updateStatusReducer, initialState);
+  const [file, setfile] = useState<File>();
+  const [contentType, setcontentType] = useState<string>("");
+  const [uploadPercent, setUploadPercent] = useState(0);
+  const [uploadStarted, setUploadStarted] = useState<boolean>(false);
+  const [uploadDone, setUploadDone] = useState<boolean>(false);
+  const [reload, setReload] = useState<boolean>(false);
```

Comment:
i am not getting any special purpose of `reload` state, can you clarify what is its need?

Reply:
i think there is no use of `reload` state as its value is changing but never used anywhere except in `setReload`